index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css"
+  integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg=="
+  crossorigin="anonymous" />
+  <title>Student Grade Table</title>
+</head>
+<body>
+  <header class="mx-auto w-50">
+    <h1>Student Grade Table</h1>
+  </header>
+  <div class="table-responsive ">
+    <table class="table table-striped table-bordered w-50 mx-auto ">
```

Comment:
The `table` class is unnecessary here. Bootstrap only uses `.table` as a way to style non-`<table>` elements like a table.